connectors/src/connectors/notion/temporal/worker.ts

```diff
@@ -44,7 +44,7 @@ export async function runNotionGarbageCollectWorker() {
     connection,
     reuseV8Context: true,
     namespace,
-    maxConcurrentActivityTaskExecutions: 8,
+    maxConcurrentActivityTaskExecutions: 3,
```

Comment:
I would leave 8 here but reduce the number of activities in the gc workflow.
(most workspace have very few pages to delete, since one workflow = one workspace, i would reduce the effect of heavy workspace but let small workspace go fast on the side)

Reply:
Ok i see your point, too bad I saw it after merging. Fixing